### ISSUE TITLE
Add suppress warning in generated l10n.dart file

### ISSUE THREAD

**Issue by narumishi** [localizely/flutter-intl-intellij#23]
Sometimes, we don't use lowerCamelCase style in .arb. For example, we may use `say_hello` or `SayHello` rather than `sayHello`.
Since our edit in generated `l10n.dart` will not be saved, it's better to add some suppress warnings at the top of file. Otherwise, Dart Analysis in Android Studio will always raise warnings. 
```
// ignore_for_file: non_constant_identifier_names
// ignore_for_file: camel_case_types
// ignore_for_file: prefer_single_quotes
```
These three warnings are ignored in another intl pulgin `Flutter i18n`.

**Comment by aleksakrstic** [comment]
Thanks @narumishi 
Seems many developers practice underscore and camelCase at the same time, so I believe we could include this in one of upcoming plugin releases.

**Comment by aleksakrstic** [comment]
@narumishi 
This is now available in version `1.8.0`
